astrology.lic

```diff
@@ -39,6 +51,67 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
+  def check_moons
+    %w(xibar yavash katamba).each { |moon| bput("pow #{moon}", 'roundtime'); waitrt? }
+  end
+
+  def check_pools
+    pools_to_size = {
+      /You have no understanding of the celestial influences over/ => 0,
+      /You have a feeble understanding of the celestial influences over/ => 1,
+      /You have a weak understanding of the celestial influences over/ => 2,
+      /You have a fledgling understanding of the celestial influences over/ => 3,
+      /You have a modest understanding of the celestial influences over/ => 4,
+      /You have a decent understanding of the celestial influences over/ => 5,
+      /You have a significant understanding of the celestial influences over/ => 6,
+      /You have a potent understanding of the celestial influences over/ => 7,
+      /You have a insightful understanding of the celestial influences over/ => 8,
+      /You have a powerful understanding of the celestial influences over/ => 9,
+      /You have a complete understanding of the celestial influences over/ => 10
+    }
+    bput('predict state all', 'roundtime')
+    lines = reget(50).reverse
```

Comment:
There's probably a more idiomatic way to do this (e.g. `map` instead of assigning to vars multiple times). I'd have to poke at it a bit to give some code. @kamarcum or @rcuhljr ?